WISE-Community/WISE
# Styled create run, share run, and share project dialogs

To test:
- Log in as teacher
- Select "Share" from the run list menu for some runs in the Class Schedule. Make sure sharing runs works correctly.
Select "Share" from the project details menu for some projects in Browse WISE Units. Make sure sharing projects works correctly.
